src/Altinn.App.Core/Infrastructure/Clients/Notifications/Telemetry.cs

```diff
@@ -0,0 +1,55 @@
+using System.Diagnostics;
+using Microsoft.ApplicationInsights;
+using Prometheus;
+
+namespace Altinn.App.Core.Infrastructure.Clients.Notifications;
+
+internal static class Telemetry
+{
+    internal static readonly Counter OrderCount = Metrics
+        .CreateCounter("altinn_app_notification_order_request_count", "Number of notification order requests.", labelNames: ["type", "result"]);
```

Comment:
Are we with this able to break the metrics down by service owner and app?

Reply:
I would guess those metadata are added when collecting the metrics?

Reply:
Could be, I'm not sure.

Reply:
Do you know where to find the metrics? I looked at the tt02 cluster at the `kube-prometheus-stack` Prometheus config and there is no configuration for externalLabels, where you would typically find this (if labels are added after/during scraping). But find for example `altinn_app_instances_created` in neither Grafana nor App Insights in Azure